### PR TITLE
Task#4600

### PR DESCRIPTION
**Pull request by zaharr2**

### CODE REVIEW

File: app/Http/Controllers/cabinet/CabinetController.php [hunk 1]
```diff
@@ -61,6 +61,17 @@ public function showMyResumes($id, Guard $auth)
         }
     }
 
+    // show all user vacancies when add new project
+    public function showAddVacancies()
+    {
+        $id = auth()->user()->id;
+        $currentuser = User::find($id);
```
Comment (Quicks) [inline app/Http/Controllers/cabinet/CabinetController.php]: You can get current_user simple auth()->user

File: app/Http/routes.php [hunk 2]
```diff
@@ -197,6 +197,8 @@
 {
     Route::get('myresumes/{id}',['as' => 'cabinet.my_resumes' ,'uses' => 'cabinet\CabinetController@showMyResumes']);
     Route::get('myvacancies/{id}',['as' => 'cabinet.my_vacancies' ,'uses' => 'cabinet\CabinetController@showMyVacancies']);
+    Route::get('project/myaddvacancies',['as' => 'cabinet.my_addvacancies' ,'uses' => 'cabinet\CabinetController@showAddVacancies']);
+    Route::get('project/myaddvacancy/{id}',['as' => 'cabinet.my_addvacancy' ,'uses' => 'Vacancy\VacancyController@getVacancy']);
```
Comment (Quicks) [inline app/Http/routes.php]: Do we need this route?

File: resources/views/project/partials/form/_vacancy.blade.php [hunk 3]
```diff
@@ -66,6 +66,15 @@ class="form-control"
...
-<div @click="addVacancy" style="color: #f76533; text-decoration:underline; cursor:pointer">{{ trans('project/description.projectAddVacancy') }} +</div>
-<br><br>
+
+<div class="btn-group dropup">
+    <button type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        Додати вакансію
+    </button>
+    <ul class="dropdown-menu">
+        <li style="background: #9d9d9d; padding-left: 3px;">Обрати з існуючих</li>
+        <li onmouseover="this.style.backgroundColor='#81a2be'" onmouseout="this.style.backgroundColor=''" v-for="userVacancy in userVacancies" :value="userVacancy.id" v-on:click="getUserVacancy" class="dropdown-item" type="button" style="padding-left: 3px;">   @{{ userVacancy.position }}</li>
```
Comment (Quicks) [inline resources/views/project/partials/form/_vacancy.blade.php]: we can set style for mouseover by css
also move styles to css

File: app/Http/routes.php (hunk 2 above)
Comment (zaharr2) [reply app/Http/routes.php]: I think yes. Otherwise how to get the right json?

**Pull request merged**